regolith/helpers/a_expensehelper.py

```diff
@@ -103,81 +118,61 @@ def db_updater(self):
         else:
             pdoc = {}
 
+
         pdoc.update({
-            'begin_date': now,
-            'log_url': '',
-            'name': rc.name,
-            'pi_id': rc.pi_id,
-            'lead': rc.lead,
-        })
-        if rc.lead is "tbd":
-            pdoc.update({
-                'status': 'proposed'
-            })
+            "begin_year": int(begin_date.split('-')[0]),
```

Comment:
just load the dates, don't split (this is a new pattern that it can be in date form or day/month/year form)

Reply:
Ok, thanks for the pointer.